lib/src/Base/Stat/LowDiscrepancySequenceImplementation.cxx

```diff
@@ -381,9 +381,9 @@ Unsigned64BitsInteger LowDiscrepancySequenceImplementation::GetNextPrimeNumber(c
 #ifdef OPENTURNS_HAVE_PRIMESIEVE
   primesieve::iterator it;
 #if PRIMESIEVE_VERSION_MAJOR >= 11
-  it.jump_to(n - 1);
+  it.jump_to(n);
```

Comment:
Could you change the commit title ? the "closes #..." should be in the message body
A rebase is needed anyway

Reply:
Ok. Fixed and rebased.